.env.example

```diff
@@ -1,33 +1,54 @@
-# Config for bot behavior. Leave this as-is unless
-# you're setting up the bot for production.
+###############################################################################
+# Configure bot behavior. This determines whether the bot will actually try to
+# post to Reddit.
+###############################################################################
 
-BOT_DRY_RUN=1
+# For development:
+#BOT_DRY_RUN=1
 
-# Config to enable praw to access Reddit. See
+# For production:
+BOT_DRY_RUN=0
```

Comment:
If you commented out MemeEconomy on line 32, shouldn't we then set `BOT_DRY_RUN` to 1 instead of 1?

Reply:
I mean, by default set all variables in examples to development mode

Reply:
Whoops, good catch! I'll fix it.